examples/go_extension/goext.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2015 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/cloudwan/gohan/extension/goext"
+	"github.com/cloudwan/gohan/extension/golang"
+)
+
+var globalEnv *goext.Environment
+
+func handleEvent(context goext.Context, resource goext.Resource) {
```

Comment:
I was expecting to see modifying the resource in the handler and passing it further